shared/Microsoft.Azure.Devices.Shared/AuthenticationType.cs

```diff
@@ -28,4 +28,4 @@ public enum AuthenticationType
         /// </summary>
         [EnumMember(Value = "certificateAuthority")] CertificateAuthority = 2
     }
-}
\ No newline at end of file
+}
```

Comment:
This file got moved meaning that the public API has changed. 
This is a breaking binary-level compat change. 

To ensure existing binary applications won't break we could add a [TypeForward](https://docs.microsoft.com/en-us/dotnet/framework/app-domains/type-forwarding-in-the-common-language-runtime).